### ISSUE TITLE
Connection errors

### ISSUE THREAD

**Issue by pachecopaulo** [zileo-net/logback-logdna#47]
I see lots of 403 errors thrown by the LogDnaAppender.

```
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,663 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,664 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,664 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,665 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,666 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,666 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,667 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,668 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,668 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,669 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond
Sep 15 16:12:17 app ERROR 2021-09-15 14:12:16,670 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: org.apache.http.NoHttpResponseException: logs.logdna.com:443 failed to respond

2021-09-15 14:59:00,612 ERROR  net.zileo.logback.logdna.LogDnaAppender: 154 - Error calling LogDna : RESTEASY004655: Unable to invoke request: javax.net.ssl.SSLException: Connection reset

```

I have also increased the connect timeout to check whether that would make a difference, but it didn't.
```
<connectTimeout>6000</connectTimeout>
<readTimeout>10000</readTimeout>
```

Any tips on how to fix this ?

**Comment by jlannoy** [comment]
Hello.

Sorry I didn't see a notification for your issue :(
Did the problem happened for a long time? It could have been a temporary outage of LogDna. I've introduced an "auto stop logging when too much error" process in the last version. But it was a quick thing, it doesn't start itself again.

I'm facing a lot of errors since several weeks too with LogDNA. 
> Error calling LogDna : Error during Accounts.find (500)
I asked for support, first they refused to help because I was using a third-party library and I had to check with the maintainer.
Then I said I was the maintainer. They looked up and told me I was logging on a not updated node on their side and it should have been fixed. Two days after that, I was having issues again. 

Regards,

Jonathan
